src/indexing/Elsa.Indexing.Elasticsearch/ElasticsearchInitialiser.cs

```diff
@@ -0,0 +1,66 @@
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+
+using Elsa.Indexing.Models;
+using Elsa.Indexing.Services;
+using Elsa.Services;
+
+using Nest;
+
+namespace Elsa.Indexing
+{
+    public class ElasticsearchInitialiser : IStartupTask
```

Comment:
Small issue with casing (should be ElasticSearch).
Furthermore, we are using American English, which means we should write it as Initializer (with a 'z').

Reply:
I have been using `Elasticsearch` everywhere. They also lowercase the 's' (https://www.nuget.org/packages/Elasticsearch.Net/). The 'z' is in the next commit.

Reply:
Ah, then I stand corrected 👍